upup/models/cloudup/resources/addons/karpenter.sh/k8s-1.19.yaml.template

```diff
@@ -1,656 +1,688 @@
-# helm template karpenter oci://public.ecr.aws/karpenter/karpenter-crd \
-#  --version v0.31.3
-# helm template karpenter oci://public.ecr.aws/karpenter/karpenter \
-#  --version v0.31.3 \
+#helm template karpenter oci://public.ecr.aws/karpenter/karpenter \
```

Comment:
Awesome!  Did you actually generate it this way?  If so I'd be in favor of adding a script here to regenerate the yaml (can be a separate PR)

Reply:
I think some of these are still manual, but I can look into it.